### ISSUE TITLE
Avoid path clashes

### ISSUE THREAD

**Issue by hinerm** [imagej/imagej.github.io#63]
Two potential sources of path conflicts:
* Other repositories in the ImageJ org that have GitHub pages (e.g. `https://github.com/imagej/list-of-update-sites` > `https://imagej.github.io/list-of-update-sites`)
* ImageJ 1 pages (e.g. `imagej.nih.gov/tutorials` > `imagej.github.io/tutorials`)

For the ImageJ1 content, we should start mirroring it to be under the `/ij` path, to mimic the paths on `imagej.nih.gov`

For other GitHub pages, we may just need to scrape the org, identify the repos with pages, and automatically detect pages that would clash. The pages should be renamed.

**Comment by ctrueden** [comment]
Here are all the imagej repos with gh-pages branches:

* conference
* list-of-update-sites
* mbf
* presentations
* tickets
* tutorials
* workshops

What if we added all of these as folders underneath `_pages` here, and put a file `DO-NOT-USE` with contents:
```
This directory is served from a dedicated repository in the imagej organization. DO NOT PUT ANY CONTENT HERE, because they will not be served on the site.
```

This would help everyone avoid falling into this trap.

We could also consider moving, on a case-by-case basis, content from the above repositories' `gh-pages` branches into this repository. But it's probably not urgent.

**Comment by ctrueden** [comment]
Resolved with 21d59081514aec30ee2f307ddf7052fc0c23e171, which marks the forbidden directories accordingly, and adds redirects for ImageJ1 content to the redirects list. The only remaining clash is `/images`, which will be addressed by #83.
